jdbc/src/main/java/org/apache/iotdb/jdbc/Utils.java

```diff
@@ -18,13 +18,20 @@
  */
 package org.apache.iotdb.jdbc;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.DateTimeException;
+import java.time.ZoneId;
+import java.time.zone.ZoneRulesException;
 import java.util.Properties;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 /** Utils to convert between thrift format and TsFile format. */
 public class Utils {
 
+  private static final Logger logger = LoggerFactory.getLogger(Utils.class);
```

Comment:
Don't add logger in SDK, as the user may be confused by the output, and there's no way to silent the log. I think it should throw a SQLException directly.

Reply:
done